cogs/obliterate.py

```diff
@@ -254,6 +255,68 @@ async def on_error(self, interaction: discord.Interaction, error: Exception):
         print(error)
         traceback.print_tb(error.__traceback__)
 
+class AppreciationModal(discord.ui.Modal, title="Message of Appreciation"):
+    def __init__(self, bot, appreciate_players):
+        super().__init__()
+        self.bot = bot
+        self.appreciate_player = appreciate_players
+
+    message = discord.ui.TextInput(label='Message of appreciation', min_length=1, max_length=200, required=True, style=TextStyle.short)
```

Comment:
I would consider using `TextStyle.long` here, and allowing significantly more characters, maybe 1000?